Makefile

```diff
@@ -26,24 +26,24 @@ refresh-api:
 	$(DOCKER_RUN) refresh_api.sh
 
 seed-fresh:
-	$(DOCKER_EXEC) "php artisan migrate:fresh --seed"
+	$(DOCKER_API) "php artisan migrate:fresh --seed"
 
 migrate:
-	$(DOCKER_EXEC) "php artisan migrate"
+	$(DOCKER_API) "php artisan migrate"
 
 artisan:
-	$(DOCKER_EXEC) "php artisan $(CMD)"
+	$(DOCKER_API) "php artisan $(CMD)"
 
 watch:
 	$(DOCKER_PNPM) run watch
 
 lint:
-	$(DOCKER_EXEC) "php ./vendor/bin/pint"
+	$(DOCKER_API) "php ./vendor/bin/pint"
 	$(DOCKER_PNPM) lint
 	docker-compose run -w /var/www/html --rm maintenance pnpm run watch
```

Comment:
While you are in here, running watch seems like a mistake here since the command is just titled "lint"

Reply:
Sure https://github.com/GCTC-NTGC/gc-digital-talent/pull/10773/commits/cf352a10632af47db2df4996cde304ce659a3510